docs/identity-platform/certificate-credentials.md

```diff
@@ -54,7 +54,7 @@ The signature is computed by applying the certificate as described in the [JSON
 {
   "alg": "PS256",
   "typ": "JWT",
-  "x5t#S256": "A1bC2dE3fH4iJ5kL6mN7oP8qR9sT0u"
+  "x5t#S256": "A1bC2dE3fH4iJ5kL6mN7oP8qR9s"
```

Comment:
This is now only 18 bytes, which is neither SHA-1 nor SHA-256 hash length. This should use a fully-formed value to prevent confusion.

```suggestion
  "x5t#S256": "EqYfThc_s6EcBdZHH3Ryj3YjG0pfzZZnzvOvh6OuTcI"
```